src/connectors/bandcamp.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 const filter = new MetadataFilter({
-	all: MetadataFilter.removeZeroWidth
+	all: MetadataFilter.removeZeroWidth,
+	all: MetadataFilter.trim,
```

Comment:
I meant the default filter defined in `filter.js`. You still have two trim calls.